app/views/configurations/index.html.erb

```diff
@@ -2,9 +2,14 @@
 
 <div class="row-fluid">
   <div class="span7 offset1">
-    <%= form_tag configurations_path do %>
-      <% @configs.each do |configuration| %>
-        <%= render partial: configuration[:type], object: configuration %>
+    <% if @configs.count == 0 %>
+      <h2>This page intentionally left blank</h2>
```

Comment:
I'd use something like "No configurable add-ons" or something more informative about the current situation.

Other examples:
https://github.com/dradis/dradis-ce/blob/master/app/views/issues/_empty.html.erb#L1
https://github.com/dradis/dradis-ce/blob/master/app/views/methodologies/index.html.erb#L29